packages/node_modules/@webex/plugin-device-manager/sample/index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Sample: Device Manager Kitchen Sink</title>
+  <title>Sample: Device Manager</title>
```

Comment:
Do we need to `git mv` this folder to the samples folder?

Reply:
Actually, yes. I'll open a separate PR to do so.